apis/v1alpha1/gatewayclass_types.go

```diff
@@ -150,11 +147,18 @@ type RouteNamespaces struct {
 	OnlySameNamespace bool `json:"onlySameNamespace"`
 }
 
-// GatewayClassParametersObjectReference identifies a parameters object for a
-// gateway class within a known namespace.
+// GatewayClassParametersObjectReference identifies a cluster-scoped parameters
+// resource for a GatewayClass.
 //
 // +k8s:deepcopy-gen=false
-type GatewayClassParametersObjectReference = ConfigMapsDefaultLocalObjectReference
+type GatewayClassParametersObjectReference struct {
+	// Group is the group of the referent.
+	Group string `json:"group" protobuf:"bytes,1,opt,name=group"`
```

Comment:
@robscott please remove the `protobuf` tags from `Group`, `Resource` and `Name` fields. Otherwise, LGTM.

Reply:
🤦 woops, thanks @danehans!